ChemoTracker/components/Profile/ProfileDetailsTable.js

```diff
@@ -1,18 +1,74 @@
 import React, { Component } from 'react';
-import { Text, View, Image } from 'react-native';
+import { Text, View, Image, ScrollView } from 'react-native';
 import { TabNavigator } from 'react-navigation';
-import Cell from './Cell.js';
-import styles from '../../styles/main.js';
+import CategoryCell from './Cell.js';
+import styles from '../../styles/profile-main.js';
 import color from '../../styles/color.js';
 
 class ProfileDetailsTable extends Component {
+  constructor(props) {
+    super(props);
+    this.peronalDetailsData = {
+      datap: [{
+        title: "Full Name",
+        description: "Carrie Bradshaw",
+        inputType: "text",
+      }, {
+        title: "Gender",
+        description: "Female",
+        inputType: "dropdown",
+      },{
+        title: "Date of Birth",
+        description: "02/04/1968",
+        inputType: "date",
+      }],
+      datac: [{
+        title: "Phone Number",
+        description: "123-456-7890",
+        inputType: "text",
+      }, {
+        title: "Email",
+        description: "cbradshaw@gmail.com",
+        inputType: "text",
+      }],
+      dataa: [{
+        title: "Allergen",
+        description: "Peanuts",
+        inputType: "text",
+      }, {
+        title: "Reaction",
+        description: "Hives",
+        inputType: "text",
+      }],
+      datam: [{
+        title: "Medical Conditions",
+        description: "High Blood Pressure",
+        inputType: "text",
+      }, {
+        title: "Medication",
+        description: "Lipitor",
+        inputType: "text",
+      }],
+      datacr: [{
+        title: "Diagnosis",
+        description: "Breast Cancer",
+        inputType: "modaldropdown",
+      }, {
+        title: "Chemotherapy",
+        description: "None",
+        inputType: "text",
+      }]
+    }
+  }
   render() {
     return (
-      <View style={styles.profileTable}>
-        <Cell title = "Full Name:" description = "Carrie Bradshaw"/>
-        <Cell title = "Diagnosis:" description = "Breast Cancer"/>
-        <Cell title = "Drug Allergies:" description = "None"/>
-      </View>
+      <ScrollView style = {styles.profileTable} pagingEnabled={true} showsVerticalScrollIndicator={true}>
```

Comment:
for the categories, you can just put them in an array and use map to loop through it
ar = [{"title:....., "data: this.per....}, {..},...{}]
ar.map(item => {
return(
<CategoryCell sectionTitle={item.title} data={item.data} />
)}